sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2022-2023.bpmn

```diff
@@ -483,516 +519,423 @@ studentCalculatedTotalIncome</bpmn:text>
+    <bpmn:sequenceFlow id="Flow_04d0tld" sourceRef="Event_0jmbywn" targetRef="Gateway_04hunux" />
+    <bpmn:sequenceFlow id="Flow_1kzoimw" name="Course Load 40 And Up" sourceRef="Gateway_1wo2y47" targetRef="Event_14d0375">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=offeringCourseLoad &gt;= 40</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_10dvc27" sourceRef="Event_14d0375" targetRef="Gateway_1yo12tx" />
+    <bpmn:sequenceFlow id="Flow_0x4ck3r" name="Course Load Less than 40" sourceRef="Gateway_1wo2y47" targetRef="Event_12swz45">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=offeringCourseLoad &lt;= 39</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_1e0u54i" sourceRef="Event_12swz45" targetRef="Gateway_1yo12tx" />
+    <bpmn:sequenceFlow id="Flow_1xoju8l" sourceRef="Gateway_1yo12tx" targetRef="Event_1otxsj1" />
+    <bpmn:sequenceFlow id="Flow_1dbuo7u" sourceRef="Event_19hr2p7" targetRef="Gateway_1wo2y47" />
+    <bpmn:sequenceFlow id="Flow_0apjmd6" sourceRef="Event_0nkkesr" targetRef="Event_1h2w0hl" />
+    <bpmn:sequenceFlow id="Flow_0foajxm" sourceRef="Event_1otxsj1" targetRef="Event_1xl9u43" />
+    <bpmn:sequenceFlow id="Flow_111mk01" sourceRef="Gateway_05k2lrk" targetRef="Event_1eaa23y" />
+    <bpmn:sequenceFlow id="Flow_1dxjc84" sourceRef="Event_1eaa23y" targetRef="Event_13hierd" />
+    <bpmn:sequenceFlow id="Flow_1rnodvu" sourceRef="Event_13hierd" targetRef="Event_19hr2p7" />
+    <bpmn:sequenceFlow id="Flow_0ehfqsk" sourceRef="Event_10par1m" targetRef="Gateway_05k2lrk" />
+    <bpmn:sequenceFlow id="Flow_147hah7" sourceRef="Event_1iix0z3" targetRef="Gateway_05k2lrk" />
+    <bpmn:sequenceFlow id="Flow_0luqxk1" name="3 or more dependants" sourceRef="Gateway_04hunux" targetRef="Event_10par1m">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=calculatedDataTotalEligibleDependants &gt;= 3</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_1lswqbt" name="Less than 3 dependants" sourceRef="Gateway_04hunux" targetRef="Event_1iix0z3">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=calculatedDataTotalEligibleDependants &lt;= 2</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_1p7fyev" sourceRef="Gateway_03c2fh6" targetRef="Activity_0pvobml" />
+    <bpmn:scriptTask id="Activity_01bf5sp" name="Partner Information and Income">
       <bpmn:extensionElements>
+        <zeebe:script expression="={&#10;  calculatedDataRelationshipStatus: appealsPartnerInformationAndIncomeAppealData.relationshipStatus,&#10;  calculatedDataPartner1TotalIncome: (if appealsPartnerInformationAndIncomeAppealData.relationshipStatus = &#34;married&#34; then appealsPartnerInformationAndIncomeAppealData.partnerEstimatedIncome else null)&#10;}" resultVariable="calculatedDataStudentPartnerInformationAndIncome" />
```

Comment:
In the below logic, is the `if` required? Would it not be consistent with the selected form option?
```
{
  calculatedDataRelationshipStatus: appealsPartnerInformationAndIncomeAppealData.relationshipStatus,
  calculatedDataPartner1TotalIncome: (if appealsPartnerInformationAndIncomeAppealData.relationshipStatus = "married" then appealsPartnerInformationAndIncomeAppealData.partnerEstimatedIncome else null)
}
```

Reply:
Yes, got it, but the check does not need to be based in the "relationshipStatus = "married", right?
The check based in the `relationshipStatus` would be a "business check" while the `is defined` with be a "technical check". I would prefer the last one for this appeals area. Does it make sense?